pwndbg/commands/version.py

```diff
@@ -146,6 +147,7 @@ def bugreport(run_browser=False, use_gh=False):
     all_info = all_versions()
 
     current_setup = "Platform: %s\n" % platform()
+    current_setup += "OS:       %s\n" % system()
```

Comment:
This provides info like 'Linux' or 'Darwin'; it would be great to display the distro name on linux :/

We should do sth like:
```suggestion
    current_setup += "OS:       %s\n" % getattr(platform, "linux_distribution", platform.system)
```

Reply:
Apologies, the ticket didn't mention distribution so I didn't know that was what was meant. 

Unfortunately platform.linux_distribution was deprecated in python3.5 and removed in python3.8.

There is platform.freedesktop_os_release(), however it is only available in python3.10 

Other than those two, there doesn't seem to be much functionality left in the platform module to determine distribution. 

I have altered amended the PR with a workaround.